lxd/storage/drivers/volume_test.go

```diff
@@ -32,14 +32,6 @@ func Test_Volume_ConfigSizeFromSource(t *testing.T) {
 			err:    nil,
 			size:   "",
 		},
-		{
```

Comment:
I don't think this should be removed. This seems wrong. The size used for instance volumes should use the pool's `volume.size` if the instance's expanded config doesn't specify any particular size.

Its just we don't store that in the instance's volume DB record config.

Does your new `ExpandedSize` solve this?

Reply:
I've reverted this test.